src/bioregistry/resource_manager.py

```diff
@@ -807,6 +807,7 @@ def rasterized_resource(self, resource: Resource) -> Resource:
             example_extras=resource.example_extras,
             example_decoys=resource.example_decoys,
             uri_format=resource.get_uri_format(),
+            rdf_uri_format=resource.get_rdf_uri_format(),
```

Comment:
I like the symmetry with `uri_format`. What are your ideas about URI versus IRI? Or is any IRI also allowed as value anyway?

Reply:
I am not familiar with places where the distinction between URI and IRI makes a difference. If we can identify a place where this matters, we can reinvestigate this question

Reply:
If it support IRIs, then it can just say IRI. That's my thinking. More precise, more clear. But this can easily be explained in the field description too. Totally fine with that.

Reply:
Done in 0b2d923e